src/app/js/user/index.js

```diff
@@ -649,6 +649,16 @@ export const getExportHiddenResources = state => {
     });
 };
 
+export const getImportHiddenResources = (state, formData) => {
+    const req = getRequest(state, {
+        url: '/api/hiddenResource/import',
+        method: 'POST',
+    });
+    delete req.headers['Content-Type'];
```

Comment:
I could set Content-Type header to `multipart/form-data` but it leads to the error `Missing boundary in multipart/form-data error` in backend.
So I delete Content-Type header to let the browser deal with it.